.github/workflows/upload-release-to-plugin-repo.yml

```diff
@@ -95,11 +112,13 @@ jobs:
                   path: ./changelog.txt
 
     upload:
-        name: Upload Gutenberg Plugin
+        name: Publish as trunk (and tag)
```

Comment:
As a minor improvement, I changed the name to something that more specifically describes what the task does in relation to the new twin task that has been added below (one uploads to `trunk`, and tags (under `tags`), the other upload it as a branch (under `branches`) and tags.